src/css/layout/header.scss

```
@@ -1,2 +1,137 @@
 @use '../object/utility/utility.scss';
-// headerのスタイルを記述するファイル
\ No newline at end of file
+// headerのスタイルを記述するファイル
+
+.l-header {
+  position: sticky;
+  top: 0;
+  left: 0;
+  padding: 25px 100px 25px 0;
+  background-color: utility.$white;
+  @include utility.tab {
+    padding: .25rem .5rem .25rem 0;
+  }
+  @include utility.sp {
+    padding: .16rem .15rem;
+  }
+  &__pc-items {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    @include utility.sp {
+      display: none!important;
+    }
+  }
+  &__lists {
+    display: flex;
+    font-size: 16px;
+    @include utility.tab {
+      font-size: .16rem;
+    }
+  }
+  &__list {
```

[こちら](https://github.com/gizumo-education/front-coding/pull/77/commits/11c4359d1f430e7d7de5d1d0d5289e6ac5c477b7)のコミットで修正していただいた内容に伴って以下のような修正が必要かと思います🙏

```suggestion
  &__items {
    display: flex;
    font-size: 16px;
    @include utility.tab {
      font-size: .16rem;
    }
  }
  &__item {
```

---

失礼！修正しました
27de7424a973c2a17b04b85df2e8cf4759e3d9ec